### ISSUE TITLE
Biome colours not being exported

### ISSUE THREAD

**Issue by Macintosh-Fan** [jmc2obj/j-mc-2-obj#200]
When I click on render biomes (with textures enabled), the snow biome grass block and grass colour does not change.

<img width="262" alt="image" src="https://user-images.githubusercontent.com/50642934/196020097-c55bb3db-7f4f-48fb-abcd-eb034d36d6bb.png">

Using the latest version of jMc2Obj on a 1.19.2 world.

**Comment by mmdanggg2** [comment]
Yep, aware of this. It never worked that well, only worked on desert and forest biomes. Trying to get a basic version of it working in the next version.

**Comment by Macintosh-Fan** [comment]
@mmdanggg2, can I somehow use Terralith's data pack to render biomes (if yes, do I put it above or below additional texture packs?)?

https://modrinth.com/datapack/terralith
